targets/mbedos5/jerryscript-mbed/jerryscript-mbed-drivers/source/AnalogIn-js.cpp

```diff
@@ -28,12 +46,18 @@ DECLARE_CLASS_FUNCTION(AnalogIn, read) {
     CHECK_ARGUMENT_COUNT(AnalogIn, read, (args_count == 0));
 
     // Extract native AnalogIn pointer
-    uintptr_t ptr_val;
-    jerry_get_object_native_handle(this_obj, &ptr_val);
+    void *native_ptr;
```

Comment:
if you had this as `void *ptr_val;` then even less lines of code should have been touched, because you could have kept `AnalogIn* native_ptr = reinterpret_cast<AnalogIn*>(ptr_val);` et al. below. (BTW, this part of the code base is C++, where traditional C-style casting is not that popular. So we should refrain from rewriting C++ casting operators to C-style casts. Although for void*-to-Something*, a `static_cast<Something*>` should be enough, `reinterpret_cast` was needed originally because `ptr_val` was not a pointer but an integer type. More about casting in C++: http://www.cplusplus.com/doc/oldtutorial/typecasting/ )

This note applies to this line / function & all other functions & conversions.